bots/elasticsearch/util.py

```diff
@@ -62,7 +62,8 @@ def fetch_agent(pks):
                 LEFT JOIN LATERAL (
                             SELECT array_agg(source.long_title) AS sources
```

Comment:
It doesn't particularly matter for ES, especially since we filter sources by exact match, but may as well 👍